.travis.yml

```diff
@@ -25,16 +26,19 @@ addons:
 before_install:
   - gem update --system
   - npm i nsp -g
+  - npm i snyk -g
 before_script:
   - mongo --version
   - mongod --version
+  - snyk test
+# adding influxdb repository
+  - curl -sL https://repos.influxdata.com/influxdb.key | sudo apt-key add -
+  - source /etc/lsb-release
+  - echo "deb https://repos.influxdata.com/${DISTRIB_ID,,} ${DISTRIB_CODENAME} stable" | sudo tee /etc/apt/sources.list.d/influxdb.list
+  - sudo apt-get update -qq
+  - sudo apt-get install influxdb -y
 after_script:
-  - nsp audit-package
-cache:
-  apt: true
-  directories:
-    - node_modules/
-    - public/lib/
+  - nsp check
```

Comment:
This one's from `trustroots/master` and shouldn't be here. I guess now GitHub shows diff of @mrkvon's `master` branch and our `master` or somerhing weird like that.

(`audit-package` is `check` nowadays)